src/h3lib/include/polygonAlgos.h

```diff
@@ -138,15 +138,17 @@ void GENERIC_LOOP_ALGO(bboxFrom)(const TYPE* loop, BBox* bbox) {
         bbox->west = 0;
         return;
     }
-    double lat;
-    double lon;
 
     bbox->south = DBL_MAX;
     bbox->west = DBL_MAX;
     bbox->north = -1.0 * DBL_MAX;
     bbox->east = -1.0 * DBL_MAX;
+    double minPosLon = DBL_MAX;
+    double maxNegLon = -1.0 * DBL_MAX;
```

Comment:
For any case of `-1.0 * DBL_MAX` I would highly suggest changing to `-DBL_MIN`. I'm sure this is what happens in most compilers anyway, but unary minus avoids theoretical multiplication.

Reply:
Do you mean `-DBL_MAX`? Will update.

Reply:
Ya my mistake. Thanks.